scripts/utils/import_languages.py

```diff
@@ -24,6 +24,9 @@
 parser.add_argument(
     '-m', '--macos', default=False, action="store_true", dest="ismacos",
     help='Include the MacOS bundle data')
+parser.add_argument(
```

Comment:
I think we cannot do it this way. 

The thing is that `ios` does have its own `qmake` but only lacks `lupdate` and `lconvert`. I believe if we do it like this we will use the wrong `qmake`.